rllib/examples/env/recsim_recommender_system_envs.py

```diff
@@ -74,15 +74,27 @@ def iex_document_sampler_creator(env_ctx):
 def iev_user_model_creator(env_ctx):
     return iev.IEvUserModel(
         env_ctx["slate_size"],
-        choice_model_ctor=choice_model.MultinomialProportionalChoiceModel,
+        choice_model_ctor=choice_model.MultinomialLogitChoiceModel,
```

Comment:
What's the difference and why did we need to change this? In the original RecSim repo, they use:
`choice_model.MultinomialProportionalChoiceModel`.

Reply:
The difference is how to handle negative logits.

MultinomialLogitChoiceModel uses ```p(x) = exp(x) / Sum_{y in scores} exp(y)```, while
MultinomialProportionalChoiceModel uses ```p(x) = (x - min_normalizer) / sum(x - min_normalizer)```. You need to know the lower bound of your output logits before you can convert everything to be positive.
intuitively, these 2 should work similarly, like the more negative model output is, the less likely it will get clicked on.